app/controllers/api/pages_controller.rb

```diff
@@ -51,4 +94,11 @@ def shallow_errors(errors)
   def get_page
     @page = Page.find(params[:id])
   end
+
+  def set_language
+    @language ||= Language.find_by(code: params[:language])
```

Comment:
I'd like to return the custom error message to be explicit that they're searching with a language that the application doesn't support. I would interpret an 404 from this endpoint as a page not found, rather than the language not found.